magda-web-client/src/Components/Dataset/MetadataExtraction/extractExtents.ts

```diff
@@ -21,31 +24,19 @@ export function extractExtents(input, output) {
             }
 
             output.temporalCoverage = {
-                intervals: [aggregateDates(rows, headers)].filter(i => i)
+                intervals: aggregateDates(rows)
             };
             output.spatialCoverage = calculateSpatialExtent(rows, headers);
         }
     }
 }
 
-import XLSX from "xlsx";
-import { uniq } from "lodash";
-const chrono = require("chrono-node");
```

Comment:
I'm having a bit of trouble coming up with a good case for a regression because it seems that the existing version works really badly too, but surely by switching away from chrono we're losing some ability to parse dates? There's a lot of formats that chrono can do that moment can't right?